src/api/util.ts

```diff
@@ -29,3 +32,55 @@ export function toHex(num: number): string {
 export function isHex(possibleHexString: string): boolean {
   return /^0x[0-9a-fA-F]+$/.test(possibleHexString);
 }
+
+export function getBaseNftFromRaw(
```

Comment:
Can you move these to `src/util/util.ts`? I'm hoping to leave all methods in `src/api` as user-facing exports. 

Feel free to rename the other file to make it less confusing!

Reply:
done!